apps/src/javalab/JavalabEditor.jsx

```diff
@@ -268,6 +271,15 @@ class JavalabEditor extends React.Component {
     };
   };
 
+  updateFileOrder() {
```

Comment:
nit: maybe `saveFileOrder`? `update` to me sounds like it's changing the order

Reply:
The updateFileOrder function calls on the action SOURCE_FILE_ORDER_UPDATED which does update the tabOrder of the files in sources according to the orderedTabKeys. 
I think I may also opt to be consistent with the other function names that are similar in function (such as updateVisibility, updateFileOrder, updateValidation) also call on actions that update and save the values within sources.

Reply:
Ahh got it, makes sense!